packages/documentation/pages/foundations/colors/list-of-tokens.vue

```diff
@@ -39,7 +39,7 @@ import {
 } from '@3yourmind/kotti-ui/tokens'
 
 export default {
-	name: 'NewColors',
+	name: 'ListOfTokens',
```

Comment:
Should follow the naming strategy the other doc pages follow (something like `DocumentationPageFoundationsColor...`)